src/ngTouch/directive/ngClick.js

```diff
@@ -158,7 +160,7 @@ ngTouch.directive('ngClick', ['$parse', '$timeout', '$rootElement',
     event.preventDefault();
 
     // Blur focused form elements
-    event.target && event.target.blur();
+    event.target && event.target.blur && event.target.blur();
```

Comment:
@pkozlowski-opensource, this is related to the SVG issue. On Chrome v40 and iOS 8.1.3, a lot of properties on the SVG element are undefined, including `tagName